actors/builtin/paych/paych_actor.go

```diff
@@ -161,7 +161,7 @@ func (pca Actor) UpdateChannelState(rt vmr.Runtime, params *UpdateChannelStatePa
 	}
 
 	if len(sv.SecretPreimage) > 0 {
-		if !bytes.Equal(rt.Syscalls().Hash_SHA256(params.Secret), sv.SecretPreimage) {
+		if !bytes.Equal(rt.Syscalls().Hash_Blake2b(params.Secret), sv.SecretPreimage) {
```

Comment:
While renaming it, please use canonical Go style (the old one caused linter problems for us!).

`HashBlake2b(b)`, or, if you like, `Hash(enum, b)` defining a const for blake2b hash algo.